### DISCUSSION TITLE
Trying to install, but not getting there

### DISCUSSION THREAD

**Discussion by mssoeren** [fabianishere/udm-iptv#137] [Q&A]
under install i'm getting a fault under configuring UDM-IPTV

cp: cannot stat '/etc/udm-iptv.conf': No such file or directory
Created symlink /etc/systemd/system/multi-user.target.wants/udm-iptv.service → /lib/systemd/system/udm-iptv.service.
#-> ubnt-dpkg-cache configure
<-# ubnt-dpkg-cache configure

Anyone?

**Comment by mssoeren** [comment]
![image](https://user-images.githubusercontent.com/94898257/173425967-2a4292e8-f0eb-4b1b-b466-6d010eadc016.png)


seems like the file is there after all.

have tried to configure several times but can not get IPTV to work.
Telenor t-wee

**Reply by fabianishere** [reply]
What does `ls -la /etc/udm-iptv.conf` say?

**Reply by mssoeren** [reply]
lrwxrwxrwx 1 root root 29 Jun 13 19:51 /etc/udm-iptv.conf -> /mnt/persistent/udm-iptv.conf

**Reply by mssoeren** [reply]
this is what udm-iptv-diag says if it helps

![image](https://user-images.githubusercontent.com/94898257/173429598-17828bea-02e9-4129-8b58-c70a1807915a.png)

**Reply by fabianishere** [reply]
Try removing the file and recreating it. It is currently a symlink to a non-existent file.

```bash
rm /etc/udm-iptv.conf
touch /etc/udm-iptv.conf
```

**Reply by fabianishere** [reply]
Do you have a UDM SE by any chance?

**Reply by mssoeren** [reply]
Not SE version.
running v.1.12.22

**Reply by fabianishere** [reply]
Interesting, could you share the output of `mount`?

**Reply by mssoeren** [reply]
i'm not very good at linux, can share it if you tell me how to do it,
btw IPTV works now :-)

**Reply by fabianishere** [reply]
Just run the command `mount` in the UniFi OS shell, where you also executed the other commands.

**Reply by mssoeren** [reply]
ok that was too easy :-P 
![image](https://user-images.githubusercontent.com/94898257/173431690-dd876091-be7c-4233-b80d-d36b9e62225d.png)

**Reply by mssoeren** [reply]
it was working for a small time yesterday. not today. any ideas?
![image](https://user-images.githubusercontent.com/94898257/173613047-f356e4d0-4382-4b10-ab2b-f4ec7faf9235.png)

**Reply by fabianishere** [reply]
Please re-run the installer: `dpkg-reconfigure -p medium udm-iptv`

**Reply by mssoeren** [reply]
Tried that, not working :-(

**Reply by mssoeren** [reply]
screenshot from debug if it helps

![image](https://user-images.githubusercontent.com/94898257/173615809-415a6481-3c17-4176-a9b5-3abfc55a63d6.png)

**Reply by mssoeren** [reply]
this command doeas not work either, ip -4 addr show dev iptv
can that be related ?.

sorry for asking so much

**Reply by fabianishere** [reply]
Is your receiver also on VLAN 4?

**Reply by mssoeren** [reply]
Yes. Receiver is on vlan 4
